examples/bare-metal-minimum/.detection-rules-cfg.json

```diff
@@ -0,0 +1 @@
+{"debug":false, "kibana_url":"http://siem.bluebaron.local", "es_user":"elastic", "kibana_user":"elastic", "es_password":"ia572iIot1500Fmu24xeAfV4", "kibana_password":"ia572iIot1500Fmu24xeAfV4" }
```

Comment:
Maybe remove this ?

Reply:
Accidentally added everything in a previous commit